AppItunes/Detail/ViewModel/DetailViewModel.swift

```diff
@@ -9,23 +9,23 @@ import Foundation
 import UIKit
 
 struct DetailViewModel {
-    let discographies: Discography
+    let discography: Discography
     
     init(discographies: Discography) {
-        self.discographies = discographies
+        self.discography = discographies
     }
     
     var title: String {
-        return discographies.collectionName ?? discographies.artistName!
+        return discography.collectionName ?? ""
     }
     
     var thumbnail: URL? {
-        return discographies.thumbnail ?? URL(string: "http://lamiradadelreplicante.com/wp-content/uploads/2015/06/swift.jpg")
+        return discography.thumbnail ?? URL(string: "http://lamiradadelreplicante.com/wp-content/uploads/2015/06/swift.jpg")
```

Comment:
Podríamos poner las constantes en un enum privado dentro de la clase para que queden más organizadas:

```suggestion
       private enum Constants {
             static let defaultImageURL = "http://lamiradadelreplicante.com/wp-content/uploads/2015/06/swift.jpg"
       }
       
       // Y aquí quedaría tal que así
       return discography.thumbnail ?? URL(string: Constants.defaultImageURL)
```